### DISCUSSION TITLE
Installing appdaemon on Home Assistant Core LXC?

### DISCUSSION THREAD

**Discussion by julred** [tteck/Proxmox#4017] [General]
Dear all,

I just re-installed my Home Assistant Core LXC to latest Home assistant version. In previous versions, there was the possibility to install appdaemon using updater script in CLI of LXC. But this option is not available anymore. Can you please advice, how to go ahead?
BR

**Comment by MickLesk** [comment]
AppDeamons are already dead / deprecated? 

HACS only offers templates and Python scripts.
